drivers/platform/aducm3029/i2c.c

```diff
@@ -65,10 +65,12 @@ static uint8_t adi_i2c_buffer[ADI_I2C_MEMORY_SIZE]
 __attribute__((aligned (4)));
 
 /** Save the current state of the bitrate to not change it each time */
-static uint32_t last_bitrate = 0;
+#define UNINITIALIZED_BITRATE 0
```

Comment:
styling: I would put defines together upper in the file, not mixed with variables

Reply:
i moved them somewhere up